bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +288,34 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
```

Comment:
I'm not sure I like that the input rows are modified. Can we leave those untouched, and return a list that is the same size as the input rows, and each element will contain a status of the corresponding mutation? So the logic could be something simple like:

```
response = []
for each entry:
  response.append(entry.status)
return response
```

This is consistent with the Go client, for example.